test/integration/targets/include_parent_role_vars/tasks/included_by_other_role.yml

```diff
@@ -0,0 +1,32 @@
+# test code for role inclusion
+# (c) 2018, Ansible Project
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- name: for display; our current parent tree
+  debug:
+    var: ansible_parent_role_names
```

Comment:
It looks like this task can be removed without affecting the test results. Doing so will minimize test output.